### ISSUE TITLE
In options readme.md forgot to close the bracket

### ISSUE THREAD

**Issue by nil4you** [joaopereirawd/fakeLoader.js#7]
replace it to
 $("#fakeloader").fakeLoader({

```
        timeToHide:1200, //Time in milliseconds for fakeLoader disappear
        zIndex:999, // Default zIndex
        spinner:"spinner1",//Options: 'spinner1', 'spinner2', 'spinner3', 'spinner4', 'spinner5', 'spinner6', 'spinner7' 
        bgColor:"#2ecc71", //Hex, RGB or RGBA colors
        imagePath:"yourPath/customizedImage.gif" //If you want can you insert your custom image
        });
```

**Comment by AlexRebula** [comment]
Already did it myself while upgrading the main code. But the pull request is still pending.

**Comment by nil4you** [comment]
ok you can close issue now, or better close it once code is upgraded.

**Comment by AlexRebula** [comment]
If you're saying that to me, I cannot do it since I am not the author and owner of this project. :)

**Comment by nil4you** [comment]
ok, thanks for IN=nfo @AlexRebula, so I will close it once it's updated. So no new user wlll get misguide.

**Comment by AlexRebula** [comment]
You're right. :) I believe Joao is busy with other things. ;)

**Comment by AlexRebula** [comment]
I believe I gave too much work to Joao since I made quite a few, more or less, significant changes, so it's understandable it's taking him longer to review and merge. However I look forward my change hopefully being merged with the master soon. :)

**Comment by nil4you** [comment]
yup, I think you also did hard work. :)

On Sat, Mar 14, 2015 at 7:27 PM, Aleksander <notifications@github.com>
wrote:

> I believe I gave too much work to Joao since I made quite a few, more or
> less, significant changes, so it's understandable it's taking him longer to
> review and merge. However I look forward my change hopefully being merged
> with the master soon. :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/joaopereirawd/fakeLoader.js/issues/7#issuecomment-80497767>
> .
>



-- 
With Regards,
Nirmal Bhagwani.

**Comment by AlexRebula** [comment]
Thanks. I needed those changes on my portfolio website, so I guessed why not just extend plugin options. :) Here's a video of the changes: https://www.screenr.com/user/alexrebula

**Comment by nil4you** [comment]
sorry, I can not see your changes untill it's merges in live, I think so.
Or you create it's extended one version in your repository's itself.

On Sat, Mar 14, 2015 at 9:47 PM, Aleksander <notifications@github.com>
wrote:

> Thanks. I needed those changes on my portfolio website, so I guessed why
> not just extend plugin options. :) Here's a video of the changes:
> https://www.screenr.com/user/alexrebula
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/joaopereirawd/fakeLoader.js/issues/7#issuecomment-80557127>
> .
>



-- 
With Regards,
Nirmal Bhagwani.

**Comment by AlexRebula** [comment]
You cannot see my changes? I don't think so... I believe you can track my changes here: https://github.com/joaopereirawd/fakeLoader.js/pull/6

**Comment by AlexRebula** [comment]
And you can see it here: https://github.com/joaopereirawd/fakeLoader.js/network

**Comment by nil4you** [comment]
Yup, I can see.

On Sat, Mar 14, 2015 at 9:54 PM, Aleksander <notifications@github.com>
wrote:

> And you can see it here:
> https://github.com/joaopereirawd/fakeLoader.js/network
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/joaopereirawd/fakeLoader.js/issues/7#issuecomment-80560468>
> .
>



-- 
With Regards,
Nirmal Bhagwani.

**Comment by AlexRebula** [comment]
I am also wondering why there is so many forks of this project but noone except me did a pull request. Just odd. :) I would believe that anyone who forks a project does that in order to make changes and improvements on it. :) However I don't see any other pull request but so many forks. :D

**Comment by nil4you** [comment]
that's why i suggested you to fork modified and make your own repository,
as I see it's author a too busy, that not able to comment on issues.

On Sat, Mar 14, 2015 at 10:01 PM, Aleksander <notifications@github.com>
wrote:

> I am also wondering why there is so many forks of this project but noone
> except me did a pull request. Just odd. :) I would believe that anyone who
> forks a project does that in order to make changes and improvements on it.
> :) However I don't see any other pull request but so many forks. :D
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/joaopereirawd/fakeLoader.js/issues/7#issuecomment-80563542>
> .
>



-- 
With Regards,
Nirmal Bhagwani.

**Comment by AlexRebula** [comment]
But as I undersand what you're saying it's already done. Once you fork a project you actually have your own copy of the project, isn't it? Check out  my profile. You should see this project among other my repositories. Or have I missed something? https://github.com/AlexRebula

**Comment by nil4you** [comment]
yup you have your own repository, but I know answer of your question why
this repository has more forks then your project

1) You have same name and content as this repository and mostly similar
design - So google has algorithm it knows copied data and it's never pushed
ahead in search

so you better change the name, change the css style or give more option of
types of loader so that your repository will have more forks then this
project.

Interesting ?

On Sat, Mar 14, 2015 at 10:05 PM, Aleksander <notifications@github.com>
wrote:

> But as I undersand what you're saying it's already done. Once you fork a
> project you actually have your own copy of the project, isn't it? Check out
> my profile. You should see this project among other my repositories. Or
> have I missed something? https://github.com/AlexRebula
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/joaopereirawd/fakeLoader.js/issues/7#issuecomment-80565667>
> .
>



-- 
With Regards,
Nirmal Bhagwani.

**Comment by AlexRebula** [comment]
Sorry my friend, but I don't understand your answer at all. I think I am talking about apples and you are talking about pears. :) Sort of speak. What has Google to do with anything? :) I am talking about forks of this master on Githhub. I have no interest my fork being shown on Google. This project has 34 forks but only two recorded pull requests and this was interesting to me.

**Comment by AlexRebula** [comment]
I said that Joao's master projectt has so many forks but only 2 of them wanted changes to be merged with Joao's master. Get it now? :D

**Comment by AlexRebula** [comment]
I am not sure we talk on the same frequency / same subject. :)

**Comment by AlexRebula** [comment]
Check this out: https://github.com/joaopereirawd/fakeLoader.js/network/members. This is what was interesting to me. Look all the members who forked Joao's master. I was just asking myself why almost no one did a pull request except the following users: me, "ghostboyzone and "jesusjzp". See here: https://github.com/joaopereirawd/fakeLoader.js/pulls?q=is%3Apr+is%3Aclosed

**Comment by nil4you** [comment]
ok.

On Sat, Mar 14, 2015 at 10:33 PM, Aleksander <notifications@github.com>
wrote:

> Check this out:
> https://github.com/joaopereirawd/fakeLoader.js/network/members. This is
> what was interesting to me. Look all the members who forked Joao's master.
> I was just asking myself why almost no one did a pull request except the
> following users: me, "ghostboyzone and "jesusjzp". See here:
> https://github.com/joaopereirawd/fakeLoader.js/pulls?q=is%3Apr+is%3Aclosed
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/joaopereirawd/fakeLoader.js/issues/7#issuecomment-80589115>
> .
>



-- 
With Regards,
Nirmal Bhagwani.

**Comment by AlexRebula** [comment]
:D
